apps/openmw/mwgui/dialogue.cpp

```diff
@@ -364,6 +364,7 @@ namespace MWGui
         bool sameActor = (mPtr == actor);
         mPtr = actor;
         mTopicsList->setEnabled(true);
+        mTopicsList->scrollToFirstItem();
```

Comment:
You can test this case by starting a dialogue, opening the console, click on the actor you're talking with and enter "forceGreeting".
That does not reset the topic scrollbar in vanilla MW.